clang/runtime/dpct-rt/include/math.hpp

```diff
@@ -36,6 +37,44 @@ class vectorized_binary<
 
 template <typename T> bool isnan(const T a) { return sycl::isnan(a); }
 // TODO: Need add more specialization such as bfloat16 version.
+
+template <typename T>
+inline int64_t zeroOrSignedExtent(const T val, const unsigned bit) {
```

Comment:
The const qualifier for passed-by-value is useless.

Reply:
fixed